examples/01-getting-started.ipynb

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
<p>I think this is more getting started with Merlin Models than MovieLens, so maybe "Getting Started with Merlin Models: Develop a Model for MovieLens"</p><p>If you can revise the title to prioritize Merlin Models, I think that'd be a worthwhile revision.</p><p> </p><p>Maybe sell a little more with the first sentence: "Merlin Models provides standard models for training deep learning recommender models."</p><p> </p><p>Second maybe "The library enables you to train standard models easily with the benefits of high-performance GPU acceleration and industry best practices already baked into the library."</p><p> </p><p>I can provide additional suggestions if you find anything in the preceding two suggestions tempting.</p><p> </p><p>For the learning objectives, it'll be best if we follow the pattern established by the NVDLI folks (ref the README at https://github.com/NVDLI/notebooks/tree/master/building-a-brain).  My sugg:</p><p> </p><h3>Learning objectives</h3><ul><li>Experiencing the basics for applying Facebooks DLRM model to a dataset</li><li>Understanding the steps for creating a model, training, and assessing it with Merlin Models</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I like the title.</p><p> </p><p>Re: Merlin Models provides standard models for training deep learning recommender models.</p><p>I dont know about standard models. I think Merlin Models will provide more functionality (e.g. building blocks, define your own architectures, fast dataloaders etc.). Maybe we should keep it more general? Same about the 2nd sentence.</p><p> </p><p>I am open for more suggestions. </p>

Reply:
<p>I updated the learning suggestions</p>

Reply:
Maybe it's okay to stick to the basics for the intro though? Selling one thing up front and expanding from there seems like it could be a reasonable strategy.